forms-web-app/.eslintignore

```diff
@@ -1 +1,2 @@
 node_modules
+arc/public
```

Comment:
Add newline at end of file

Reply:
`src` not `arc`

Reply:
If you don't have it, I advise you to use the plugin "prettier" which is automatically doing formatting/indentations following the good practices rules. Made my life more easier and prettier.

I don't which IDE you are using but it's available on most of them.

Reply:
Yeah, not sure why it's not working on @mmuchlinski's machine. There's an `.editorconfig` file in the root which should be working out the box on Atom as well